.meshrc.yaml

```diff
@@ -141,6 +141,15 @@ sources:
             responseTypeName: UpdateSampleNotes
             headers:
               Cookie: "{context.headers['cookie']}"
+          - type: Mutation
+            field: KickoffWorkflow
```

Comment:
This endpoint is actually called for kick off WGS and AMR (see linked code below).  Could we create two separate mutations and name them specifically, so maybe `KickoffWGSWorkflow` and `KickoffAMRWorkflow`?  Also, could we add a type for the `inputs_json` field, since they are different for each one?

Technically, we only need WGS for now, but if it's not too much lift, it'd probably make sense to go ahead and federate AMR as well since they're pretty similar.  That way we don't have to come back to this later on and built context again.

https://github.com/chanzuckerberg/czid-web-private/blob/4d6435761c594bb4214b2a5b30622e1cfc028a9d/app/assets/src/api/index.ts#L716-L744

Reply:
We don't need, or generally add types, for input arguments. That gets picked up automatically.

Reply:
It does get picked up automatically, but in this case, since the inputs are different depending on pipeline, it would be helpful to type them.

Also, nothing stops us from setting up two separate mutation fields, as I mentioned above, that call the same REST endpoint right?  When we convert to graphQL in React, we would be able to call a workflow specific mutation, rather than a generic kickoff workflow.